### ISSUE TITLE
Profile changes are not cancelled after cancel button click

### ISSUE THREAD

**Issue by ndebuhr** [hydroshare/hydroshare#2873]
Login, make changes to the profile details, then "Cancel" changes.  Upon returning to the edit interface, the cancelled edits are still there.  The system requires a logout/login cycle to clear the edits.

[Video and Other Bug Report Files](https://github.com/hydroshare/hydroshare/files/2288940/004.zip)

**Comment by engineSound** [comment]
Watched the testing video. Thanks!

I need to triage this issue.

**Comment by engineSound** [comment]
We are inclined to treat this as a front end jquery issue. Fix will be discussed with @aphelionz

**Comment by engineSound** [comment]
When cancel button is clicked, the form that has been altered is in cache doesn't reset back. This problem can be fixed by calling location.reload() go back to the profile page. I tested the change it works. @aphelionz What do you think?

**Comment by aphelionz** [comment]
I think that's a fair approach. Make a PR and I will review @engineSound

**Comment by engineSound** [comment]
PR https://github.com/hydroshare/hydroshare/pull/2925

**Comment by engineSound** [comment]
fixed and merged into develop branch.

**Comment by ndebuhr** [comment]
Please see comments in https://github.com/hydroshare/hydroshare/issues/2934.  

> I'm seeing same behavior as originally reported for the "Profile changes are not cancelled after cancel button click" issue in beta. See original bug report #2873 and use same steps to reproduce.

I'm reopening this issue so we don't lose track of the fact that this needs some additional work

**Comment by ndebuhr** [comment]
We released 1.16 despite the comment noted above.  Additional note: I'm still seeing in 1.16.1 beta deployment

**Comment by engineSound** [comment]
The fix worked on chrome and safari browser. However in Firefox browser doesn't take the effect.

**Comment by engineSound** [comment]
This one is fixed and pending on code review #3085.

**Comment by engineSound** [comment]
In 1.19 release. Close after the release is out.

**Comment by engineSound** [comment]
It didn't make into the 1.19 release. Waiting for the next minor release.

**Comment by cuahsimarko** [comment]
close me?

**Comment by Lizabrazil** [comment]
@Lizabrazil check the status
